charts/k8s-triliovault-operator/k8s-triliovault-operator/2.1.0+upv2.1.0/templates/serviceAccount.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: k8s-triliovault-operator
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: k8s-triliovault-operator
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
I'm running into the following problem when installing the chart which seems to be caused by this line. It uses the whole version including the build metadata with the `+` which is not a valid character for a label and fails.

```
Error: ServiceAccount "k8s-triliovault-operator" is invalid: metadata.labels: Invalid value: "k8s-triliovault-operator-2.1.0+upv2.1.0": a valid label must be an empty string or consist of alphanumeric characters, '-', '_' or '.', and must start and end with an alphanumeric character (e.g. 'MyValue', or 'my_value', or '12345', regex used for validation is '(([A-Za-z0-9][-A-Za-z0-9_.]*)?[A-Za-z0-9])?')
```

Resolved service account template taken from the output of `helm template assets/k8s-triliovault-operator/k8s-triliovault-operator-2.1.0+upv2.1.0.tgz`
```
apiVersion: v1                                                                                                                      
kind: ServiceAccount                                                                                                                
metadata:                                                                                                                           
  name: k8s-triliovault-operator                                                                                                    
  namespace: default                                                                                                                
  labels:                                                                                                                           
    app: k8s-triliovault-operator                                                                                                   
    chart: "k8s-triliovault-operator-2.1.0+upv2.1.0"                                                                                
    release: "RELEASE-NAME"                                                                                                         
    heritage: "Helm" 
```

Reply:
@PennyScissors I understand here that the service account issue is due to the label *chart: "k8s-triliovault-operator-2.1.0+upv2.1.0"*. But originally this label was *chart: "k8s-triliovault-operator-2.1.0* and after rancher chart generation it is updated to add *+upv2.1.0*.
So even if we update the helm chart and remove 'v' from the *version* and *appVersion* parameter values, the rancher chart script will still concatenate the *appVersion* value to *version* and this problem will arise.